packages/verification-vue-component/src/locales/de.json

```diff
@@ -37,6 +37,17 @@
                         "content": "Die Certifaction-Technologie wird von verschiedenen innovativen Anbietern genutzt, die ihre Kunden jeweils genau prüfen, bevor sie ihre Identität verifizieren. Für Details können Sie sich direkt an den Anbieter wenden (siehe \"Identität verifiziert durch\")."
                     }
                 ]
+            },
+            "contact": {
+                "title": "Support kontaktieren",
+                "form": {
+                    "email": "Ihre E-Mail",
+                    "question": "Ihre Frage",
+                    "consent": "Mit dem Absenden des Formulars akzeptiere ich die Datenschutzerklärung.",
+                    "submit": "Anmelden",
```

Comment:
"Anmelden" doesn't work here. Would replace it with "Absenden". Rest looks good

Reply:
Fixed in f3ee7a4